frontend/src/container/WorkspaceContent.jsx

```diff
@@ -58,6 +57,7 @@ import {
   setWorkspaceDetail
 } from '../action-creator.sync.js'
 import uniq from 'lodash/uniq'
+import { CONTENT_TYPE } from 'tracim_frontend_lib/src/helper.js'
```

Comment:
only import from `tracim_frontend_lib` to let the resolver do his work

Reply:
Changed